src/Core/src/Platform/Windows/MauiNavigationView.cs

```diff
@@ -180,7 +180,12 @@ internal void UpdatePaneDisplayModeFromFlyoutBehavior(FlyoutBehavior flyoutBehav
 			{
 				case FlyoutBehavior.Flyout:
 					IsPaneToggleButtonVisible = true;
-					PaneDisplayMode = NavigationViewPaneDisplayMode.LeftMinimal;
+					// Updating the same PaneDisplayMode at the runtime causes selection related issues
```

Comment:
Could expand the comment to include more details, like the link to the original issue?

```
// WinUI bug: Setting PaneDisplayMode to the same value during navigation
// causes the selected item indicator to not update correctly.
// Workaround: Only set PaneDisplayMode when the value actually changes.
// Related: https://github.com/microsoft/microsoft-ui-xaml/issues/9812
```

Reply:
yes, I have updated it.